connection.go

```diff
@@ -10,39 +10,47 @@ import (
 	"strings"
 	"time"
 
+	orkuserr "github.com/orkusinc/api/common/errors"
```

Comment:
Dont include orkus packages into vendor, as its supposed to be open source.

Reply:
Done